client/src/components/_common/Form/FormField.jsx

```diff
@@ -85,7 +85,10 @@ const FormField = ({
   const FieldNote = () => (
     <>
       {meta.touched && meta.error ? (
-        <div className="form-field__validation-error">{meta.error}</div>
+        <FormText className="form-field__help" color="muted">
+          {description}
+          <div className="form-field__validation-error">{meta.error} </div>
+        </FormText>
       ) : (
         description && (
           <FormText className="form-field__help" color="muted">
```

Comment:
We have duplicated code on line 88 and line 94 here with identical `<FormText ...`. We can simplify this by bringing the `meta.error` conditional down to wrap the `<div>` that contains the error text